packages/perseus-core/src/parse-perseus-json/perseus-parsers/legacy-button-sets.ts

```diff
@@ -0,0 +1,21 @@
+import {array, enumeration} from "../general-purpose-parsers";
+import {defaulted} from "../general-purpose-parsers/defaulted";
+
+export const parseLegacyButtonSet = enumeration(
+    "basic",
+    "basic+div",
+    "trig",
+    "prealgebra",
+    "logarithms",
+    "basic relations",
+    "advanced relations",
+    "scientific",
+);
+
+export const parseLegacyButtonSets = defaulted(
+    array(parseLegacyButtonSet),
+    // NOTE(benchristel): I copied the default buttonSets from
+    // expression.tsx. See the parse-perseus-json/README.md for
+    // an explanation of why we want to duplicate the default here.
+    () => ["basic", "trig", "prealgebra", "logarithms"] as const,
```

Comment:
So this shouldn't contain `scientific` because it represents old content and needs to not change over time to prevent old exercises from becoming "unsolvable or otherwise misconfigured," is that correct? Want to make sure I read the right part of the readme 😂